services/application/src/mongodb/schema/application.js

```diff
@@ -1,8 +1,38 @@
 const { Schema } = require('mongoose');
+const stripTags = require('striptags');
 const { organizationPlugin } = require('@identity-x/mongoose-plugins');
 const { normalizeEmail } = require('@identity-x/utils');
 const { emailValidator } = require('@identity-x/mongoose-plugins');
 
+const stripHtml = (v) => {
+  if (!v) return null;
+  return stripTags(v, ['b', 'strong', 'em', 'i', 'u']);
+};
+
+const loginLinkTemplate = new Schema({
+  subject: {
+    type: String,
+    set: (v) => {
```

Comment:
The three setters can change to `set: stripHtml,` -- since `stripHtml` is doing the exact same logic